Makefile

```diff
@@ -161,8 +161,31 @@ sub-image-%:
 	@$(MAKE) ARCH=$* image
 
 image: ## make the image for a single ARCH
-	docker buildx build --load -t $(BUILD_IMAGE):latest-$(ARCH) -f Dockerfile --platform $(OS)/$(ARCH) .
-	echo "Done. image is at $(BUILD_IMAGE):latest-$(ARCH)"
+	docker buildx build --load -t $(BUILD_IMAGE):$(BUILD_TAG)-$(ARCH) -f Dockerfile --platform $(OS)/$(ARCH) .
```

Comment:
This phrase `$(BUILD_IMAGE):$(BUILD_TAG)-$(ARCH)` is repeated in several places (4 at last count) and the one without `-$(ARCH)` in several more (also 4, I think). These should be replaced with a single var:

```makefile
TAGGED_IMAGE ?= $(BUILD_IMAGE):$(BUILD_TAG)
TAGGED_ARCH_IMAGE ?= $(TAGGED_IMAGE)-$(ARCH)
```

and then use those